services/chat/api.go

```diff
@@ -421,7 +431,8 @@ func getChatMembers(sourceChat *protocol.Chat, community *communities.Community,
 	if community != nil {
 		channel, exists := community.Chats()[sourceChat.CommunityChatID()]
 		if !exists {
-			return result, communities.ErrChatNotFound
+			// Skip unknown community chats. They might be channels that were deleted. We shouldn't get here
+			return result, nil
```

Comment:
Does it still make sense to keep this change if we already have `!exists` check in `getChannelGroups`?

Reply:
It's a double security. In case someone calls `getChatMembers` from somewhere else. But yeah, you're right that in the current context, this should never be called, hence why I added `We shouldn't get here` in the comment.